frontend/src/components/mine/ContactInfo/ViewPartyRelationships.js

```diff
@@ -139,21 +137,68 @@ export class ViewPartyRelationships extends Component {
 
     payload.start_date = values.start_date;
     payload.end_date = values.end_date;
-    payload.mine_tailings_storage_facility_guid = values.mine_tailings_storage_facility_guid;
+    payload.related_guid = values.related_guid;
 
     this.props.updatePartyRelationship(payload).then(() => {
-      this.props.fetchPartyRelationships(this.props.mine.guid);
+      this.props.fetchPartyRelationshipsByMineId(this.props.mine.guid);
       this.props.closeModal();
     });
   };
 
   removePartyRelationship = (event, mine_party_appt_guid) => {
     event.preventDefault();
     this.props.removePartyRelationship(mine_party_appt_guid).then(() => {
-      this.props.fetchPartyRelationships(this.props.mine.guid);
+      this.props.fetchPartyRelationshipsByMineId(this.props.mine.guid);
     });
   };
 
+  renderInactiveRelationships = (partyRelationships) => {
+    const activeRelationships = partyRelationships.filter(
+      (x) =>
+        x.end_date === "None" ||
+        (Date.parse(x.end_date) >= new Date() &&
+          (x.start_date === "None" || Date.parse(x.start_date) <= new Date()))
+    );
+    const inactiveRelationships = partyRelationships.filter(
```

Comment:
If they're mutually-exclusive, the `inactiveRelationships` should be the difference between the `partyRelationships` and `activeRelationships`, which could help DRY this up and make it more expressive.

Reply:
I had intended to refactor this a bit after getting it working but I never got back to it. Updated to filter based on active relationships, and to use uniqBy.